unlock-app/src/hooks/useCreditCard.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query'
+import { getFiatPricing } from './useCards'
+import { useConfig } from '~/utils/withConfig'
+
+interface CreditCardProps {
+  lockAddress: string
+  network: number
+  quantity?: number
+}
+
+export const useGetCreditCardPricing = ({
+  lockAddress,
+  network,
+}: CreditCardProps) => {
+  const config = useConfig()
+
+  return useQuery(['getCreditCardPricing', lockAddress, network], async () => {
+    const keyPricing = await getFiatPricing(config, lockAddress, network)
+    return keyPricing
+  })
+}
```

Comment:
Don't we already have a hook for this? usePricing

Reply:
I need credit card converted price here, and `usePricing` does not return this value